docs/source/projects.rst

```diff
@@ -23,13 +23,13 @@ A new project directory can be created empty, or populated using one
 or more :term:`template` plugins. Templates should be specified as
 arguments to :ref:`command-mkproject`. Multiple values can be provided
 to apply more than one template. For example, to check out a Mercurial
-repository from a project on bitbucket and create a new Django
-site, combine the :ref:`templates-bitbucket` and
+repository from a project on GitHub and create a new Django
+site, combine the :ref:`templates-github` and
 :ref:`templates-django` templates.
 
 ::
 
-    $ mkproject -t bitbucket -t django my_site
+    $ mkproject -t github -t django my_site
```

Comment:
Same here, the example template is for a bitbucket project, not GitHub, so we don't want the changes to this file.

Reply:
Somehow this wasn't reverted?

Reply:
how does one save files... fixed it